projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-menu/wpcom-admin-menu.php

```diff
@@ -148,31 +147,142 @@ function wpcom_add_hosting_menu() {
+
+	$activity_log_url = 'https://wordpress.com/activity-log/' . $domain;
+	$vaultpress_url   = 'https://wordpress.com/backup/' . $domain;
+	$monetize_url     = 'https://wordpress.com/earn/' . $domain;
+	$subscribers_url  = 'https://wordpress.com/subscribers/' . $domain;
+	$newsletter_url   = 'https://wordpress.com/settings/newsletter/' . $domain;
+	$scan_url         = 'https://wordpress.com/scan/history/' . $domain;
+
+	// Add submenu items that link to WordPress.com.
+	add_submenu_page(
+		'jetpack',
+		__( 'Activity Log', 'jetpack-mu-wpcom' ),
+		__( 'Activity Log', 'jetpack-mu-wpcom' ),
+		'manage_options',
+		$activity_log_url,
+		null // @phan-suppress-next-line PhanTypeMismatchArgumentProbablyReal -- Core should ideally document null for no-callback arg. https://core.trac.wordpress.org/ticket/52539.
+	);
+
 	add_submenu_page(
 		'jetpack',
+		__( 'VaultPress', 'jetpack-mu-wpcom' ),
+		__( 'VaultPress', 'jetpack-mu-wpcom' ),
+		'manage_options',
+		$vaultpress_url,
+		null // @phan-suppress-next-line PhanTypeMismatchArgumentProbablyReal -- Core should ideally document null for no-callback arg. https://core.trac.wordpress.org/ticket/52539.
+	);
+
+	add_submenu_page(
+		'jetpack',
+		__( 'Monetize', 'jetpack-mu-wpcom' ),
 		__( 'Monetize', 'jetpack-mu-wpcom' ),
-		__( 'Monetize', 'jetpack-mu-wpcom' ) . ' <span class="dashicons dashicons-external"></span>',
 		'manage_options',
-		esc_url( Redirect::get_url( 'calypso-monetize' ) ),
-		null // @phan-suppress-current-line PhanTypeMismatchArgumentProbablyReal
+		$monetize_url,
+		null // @phan-suppress-next-line PhanTypeMismatchArgumentProbablyReal -- Core should ideally document null for no-callback arg. https://core.trac.wordpress.org/ticket/52539.
 	);
+
+	if ( $is_atomic_site ) {
+		add_submenu_page(
+			'jetpack',
+			__( 'Scan', 'jetpack-mu-wpcom' ),
+			__( 'Scan', 'jetpack-mu-wpcom' ),
+			'manage_options',
+			$scan_url,
+			null // @phan-suppress-next-line PhanTypeMismatchArgumentProbablyReal -- Core should ideally document null for no-callback arg. https://core.trac.wordpress.org/ticket/52539.
+		);
+	}
+
+	add_submenu_page(
+		'jetpack',
+		__( 'Subscribers', 'jetpack-mu-wpcom' ),
+		__( 'Subscribers', 'jetpack-mu-wpcom' ),
+		'manage_options',
+		$subscribers_url,
+		null // @phan-suppress-next-line PhanTypeMismatchArgumentProbablyReal -- Core should ideally document null for no-callback arg. https://core.trac.wordpress.org/ticket/52539.
+	);
+
+	if ( $is_simple_site ) {
+		add_submenu_page(
+			'jetpack',
+			__( 'Newsletter', 'jetpack-mu-wpcom' ),
+			__( 'Newsletter', 'jetpack-mu-wpcom' ),
+			'manage_options',
+			$newsletter_url,
+			null // @phan-suppress-next-line PhanTypeMismatchArgumentProbablyReal -- Core should ideally document null for no-callback arg. https://core.trac.wordpress.org/ticket/52539.
+		);
+	}
+
+	// Re-order menu.
+	global $submenu;
+	if ( ! isset( $submenu['jetpack'] ) ) {
```

Comment:
Is this possible?

Reply:
Yes, if the user doesn't have the needed capabilities to see the Jetpack menu.